pkg/volume/csi/csi_plugin.go

```diff
@@ -839,9 +846,13 @@ func (p *csiPlugin) supportsVolumeLifecycleMode(driver string, volumeMode storag
 	if p.csiDriverLister != nil {
 		kletHost, ok := p.host.(volume.KubeletVolumeHost)
 		if ok {
-			if err := kletHost.WaitForCacheSync(); err != nil {
+			synced, err := kletHost.CSIDriversSynced()
+			if err != nil {
 				return err
 			}
+			if !synced {
+				return errors.New("CSIDrivers has not bean synced yet")
```

Comment:
typo: "bean" -> "been"

Reply:
Thanks for your suggestion, I have modified the code